docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.0'
 services:
   searcharr:
     container_name: searcharr
-    image: toddrob/searcharr:latest
+    build: .
```

Comment:
Why build instead of pulling the image? That will require that users clone the repo instead of just creating a docker-compose.yml file.